ram/wram.asm

```diff
@@ -1914,7 +1914,7 @@ wMissableObjectFlagsEnd::
 	ds 7
 
 ; temp copy of SPRITESTATEDATA1_IMAGEINDEX (used for sprite facing/anim)
-wd5cd:: db
+wSpriteImageIndexTempCopy:: db
```

Comment:
Why is it temporarily copied here? Is it being backed up while `wSpriteImageIndex` takes some other value?
```suggestion
wSpriteImageIndexBackup:: db
```

Reply:
It's possible updatesprites can modify the image index between where it's written and read, but it might just be so they don't have to navigate to the image index again in a 16-bit register